.github/workflows/e2e_targeted.yaml

```diff
@@ -17,6 +17,10 @@ on:
         description: 'KIC Docker image to test with. The default "localhost:5000/kong/kubernetes-ingress-controller:ci" builds an image from the dispatch branch'
         required: true
         default: 'localhost:5000/kong/kubernetes-ingress-controller:ci'
+      include-integration:
+        description: 'Set to "true" to run integration tests also'
+        required: true
+        default: 'false'
```

Comment:
It actually almost seems like we would want to turn this on by default?

Reply:
This job has two main purposes:
- Running E2E on PRs that modify E2E and/or functionality that affects E2E test behavior.
- Version compatibility testing.

Integration tests are unnecessary for the former (they're identical to the normal PR integration job), so it's there as a time-saving measure--the first case comes up more often IME.